images-list

```diff
@@ -449,6 +455,7 @@ neuvector/enforcer rancher/mirrored-neuvector-enforcer 5.0.0-b1
 neuvector/enforcer rancher/mirrored-neuvector-enforcer 5.0.0-b2
 neuvector/manager rancher/mirrored-neuvector-manager 5.0.0-b1
 neuvector/manager rancher/mirrored-neuvector-manager 5.0.0-b2
+nginx rancher/mirrored-nginx 1.21.6
```

Comment:
We are using `nginx` as the webserver frontend when epinio is deployed with an internal container registry component.
While I saw the `library/nginx` lines when making the PR I discounted them because of the `library` prefix. IOW I was unsure if that was actually an nginx webserver image or something just related to nginx, but not a server.

The using spec is
```
{{ if .Values.containerregistry.enabled }}
      - name: nginx
        image: "{{ template "registry-url" . }}{{ .Values.containerregistry.image.nginx.repository}}:{{ .Values.containerregistry.image.nginx.tag }}"
        imagePullPolicy: IfNotPresent
        securityContext:
          runAsUser: 1000
          runAsNonRoot: true
          allowPrivilegeEscalation: false
          readOnlyRootFilesystem: true
        livenessProbe:
          tcpSocket:
            port: 5000
          initialDelaySeconds: 15
          periodSeconds: 20
        readinessProbe:
          tcpSocket:
            port: 5000
        volumeMounts:
        - mountPath: /etc/nginx/conf.d
          name: nginx-conf
        - mountPath: /var/cache/nginx/
          name: nginx-run
        - mountPath: /var/run/
          name: nginx-run
{{- end }}

---
apiVersion: v1
kind: ConfigMap
metadata:
  name: nginx-conf
  namespace: {{ .Release.Namespace }}
data:
  nginx.conf: |
    server {
      listen 30500 default_server;
      server_name 127.0.0.1;

      location / {
        proxy_pass https://localhost:5000/;
      }
    }
{{- end }}

```

in case that helps to make the determination.

Reply:
@aiyengar2 :arrow_up:

Reply:
Resolved per meeting. Commit 1d0a124a257814e3f8c9de4780679e387989994f